app/component/itinerary/ItineraryDetails.js

```diff
@@ -293,16 +293,13 @@ class ItineraryDetails extends React.Component {
                   legs={itinerary.legs}
                 />
               )),
-            this.props.setNavigation && false && (
-	      // TODO: placeholder for real component, change this
+
+            this.props.setNavigation && (
               <div key="tracking">
-                <SecondaryButton
-                  buttonName="tracking"
-                  ariaLabel="tracking"
+                <ToTripButton ariaLabel="followyourouteonscreen" 
                   buttonClickAction={() => {
```

Comment:
This dedicated button does not need external arialabel prop.

Reply:
ariaLabel removed.